IntergrationTestHardwareServer.md

```diff
@@ -0,0 +1,41 @@
+Integratietesting Hardware-Server 2nd-factor
+
+- Verbinding maken en verbreken
+	Beschrijving: Een verbinding maken met de server en deze verbinding weer verbreken.
+	Verwacht resultaat: De verbinding tussen client en server wordt gemaakt en gestopt.
+	Resultaat:
+
+	
+- Meerdere verbindingen tegelijk maken
+	Beschrijving: Meerdere clients maken maken tegelijk verbinding met de server.
+	Verwacht resultaat: Meerdere verbinding bestaan langs elkaar.
+	Resultaat:
+	
+	
+- Bericht sturen open verbinding
+	Beschrijving: Bericht "Test" wordt verstuurd naar de server.
+	Verwacht resultaat: Client krijgt response "het bericht Test is afgehandeld".
+	Resultaat:
+	
+	
+- Andere netwerkpoort gebruiken
+	Beschrijving: Er wordt verbinding gemaakt met poortnummer 3000.
+	Verwacht resultaat: Een foutmelding dat het poortnummer 3000 niet open staat.
+	Resultaat:	
+	
+	
+- Rest netwerkpoort gebruiken
+	Beschrijving: Er wordt verbinding gemaakt met poortnummer 8080.
+	Verwacht resultaat: Een foutmelding dat poort 8080 niet open staat.
+	Resultaat:	
```

Comment:
Is het doel van deze test niet het zelfde. Maakt dit de test niet dubbel?

Reply:
ja in principe wel. het was een eerste ingeving. Ik wist niet eens of je die mogelijkheid moet testen om te verbinden met een andere poort